widgets/BigQueryExecute-action.json

```diff
@@ -0,0 +1,130 @@
+{
+  "metadata": {
+    "spec-version": "1.5"
+  },
+  "display-name": "Google BigQuery Execute",
```

Comment:
source and sink use 'BigQuery' not 'Google BigQuery', let's do the same here: 'BigQuery Execute'